packages/graphql-async/graphql-async.0.9.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "Andreas Garnaes <andreas.garnaes@gmail.com>"
+authors: "Andreas Garnaes <andreas.garnaes@gmail.com>"
+homepage: "https://github.com/andreas/ocaml-graphql-server"
+doc: "https://andreas.github.io/ocaml-graphql-server/"
+bug-reports: "https://github.com/andreas/ocaml-graphql-server/issues"
+dev-repo: "git+https://github.com/andreas/ocaml-graphql-server.git"
+
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {build}
+  "graphql" {>= "0.9.0"}
+  "async_kernel" {>= "v0.9.0"}
+  "alcotest" {with-test}
+  "async_unix" {with-test & >= "v0.9.0"}
```

Comment:
It's unclear to me whether I should keep the upper bound constraint on `async_kernel` and `async_unix` which was added to `graphql-async.0.8.0` in [this PR](https://github.com/ocaml/opam-repository/pull/13344).

Reply:
Ping @xclerc. I would probably keep it and relax it after the release of the new janestreet libraries

Reply:
Indeed, that is the plan.

Reply:
Ok, I’ll add the constraint later today.

Reply:
I've added the constraint now.